setup/docker-compose.setup.yml

```
@@ -0,0 +1,11 @@
+version: "3.8"
+
+services:
+  setup:
+    image: caddy:2.7.5-alpine
+    ports:
+      - "2019:2019" # Caddy Admin API - only necessary for initially trusting its CA certificate.
```

Did you intend to publicly publish these ports? Docker defaults to `0.0.0.0/0` [even if`ufw` is manually configured to block incoming connections from outside to that port](https://github.com/moby/moby/issues/4737).

```suggestion
    ports:
      - "127.0.0.1:2019:2019" # Caddy Admin API - only necessary for initially trusting its CA certificate.
```

---

@RudolfSchreier  and @devnoname120  done and tested. All works. Thanks for the suggestion.

---

Just caught up with @roosnic1 via Teams - this change is actually in #3, but he will merge both after one another, so this is fine :).